src/platform/plugins/shared/data_view_field_editor/public/components/preview/field_list/field_list_item.tsx

```diff
@@ -150,8 +150,8 @@ export const PreviewListItem: React.FC<PreviewListItemProps> = ({
         className={classes}
         // highlights the field using token, TODO: migrate whole SCSS file to emotions
         css={css`
-          background-color: ${isFromScript ? euiTheme.colors.backgroundBasePrimary : 'inherit'};
-          font-weight: ${isFromScript ? 'bold' : 'normal'};
+          background-color: ${isFromScript ? euiTheme.colors.backgroundBasePrimary : undefined};
+          font-weight: ${isFromScript ? 600 : 'normal'};
```

Comment:
```suggestion
          font-weight: ${isFromScript ? 600 : undefined};
```
Font weight looks good now, but I think it would still be best to not apply any font weight styles when `isFromScript` is false.

Reply:
Yes, sorry! Missed this one 🙈